app/decorators/service_decorator.rb

```diff
@@ -68,7 +68,7 @@ def new_application_data
   end
 
   def backends_table_data
-    BackendApiDecorator.decorate_collection(backend_apis)
+    BackendApiDecorator.decorate_collection(backend_apis.sort_by(&:name))
```

Comment:
```suggestion
    BackendApiDecorator.decorate_collection(backend_apis.reorder(name: :asc))
```

Sorry that was the thing